lyluatex.md

```diff
@@ -52,10 +52,12 @@ tlmgr install lyluatex
 
 #### Latest version
 
-Copy `lyluatex.sty` and `lyluatex.lua` from this repository into your `TEXMF`
-tree, or clone this repostory into your `TEXMF` tree using Git, then run
-`mktexlsr`. Note that in this case your local copy will shadow the version
-possibly installed in your \TeX\ distribution.
+Copy `lyluatex.sty` and `lyluatex.lua` from this repository into your
+`TEXMFHOME` tree, or clone this repostory into your `TEXMFHOME` tree using Git.
+In many cases this will be `$HOME/texmf`, and \lyluatex\ should be located below
+`$TEXMFHOME/tex/luatex`. It is important that this is the `luatex` tree because
+otherwise \LuaLaTeX\ will still find the version from TeXLive first and use that
```

Comment:
You only need one of `because` and `otherwise`.  `otherwise` is more appropriate here because you give the counter example (what will go wrong if you don't do it right).  `because` would be for actually providing the reason why to do it the right way (i.e., engine specific distro trees have higher priority than generic local ones).